app/src/pages/trace/EditSpanAnnotationsDialog.tsx

```diff
@@ -227,9 +227,7 @@ function SpanAnnotationsList(props: {
   const hasAnnotations = annotations.length > 0;
   return (
     <div>
-      {!hasAnnotations && (
-        <Empty graphicKey="documents" message="No annotations for this span" />
-      )}
+      {!hasAnnotations && <SpanAnnotationsEmpty />}
```

Comment:
I'm not sure this makes sense here. You can add the annotations in this UI. I would remove here.

Reply:
ahh good point will remove